Cargo.lock

```diff
@@ -11613,8 +11587,9 @@ dependencies = [
 
 [[package]]
 name = "sqlx"
-version = "0.8.2"
-source = "git+https://github.com/kamu-data/sqlx?branch=nextest-race-condition#c322d7c55d0546dcd723e67e123a61242d2ced19"
+version = "0.8.6"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "1fefb893899429669dcdd979aff487bd78f4064e5e7907e4269081e0ef7d97dc"
```

Comment:
All chechsums from `sqlx-*` deps have been verified manually -- the same as in `kamu-cli`